.vscode/launch.json

```diff
@@ -7,9 +7,11 @@
     {
       "type": "node",
       "request": "attach",
-      "name": "Attach",
+      "name": "Attach to Remote",
+      "address": "127.0.0.1",
       "port": 9229,
-      "skipFiles": ["<node_internals>/**"]
```

Comment:
I don't know enough about this file's config. Is there a reason you removed `skipFiles`?

Reply:
I must have added skip files while copying from elsewhere during dev.  In the launch.json in the old monorepo, this configuration did not have the `skipFiles` property present: https://github.com/forcedotcom/salesforcedx-apex/blob/be31310f9ad92e8a13e876ad98efbab65b68463f/.vscode/launch.json#L10